tests/tests.go

```diff
@@ -209,6 +210,26 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() (time.Duration, error) {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
```

Comment:
Let's start to use controller-runtime client `testClient.VirtClient` (name is very unfortunate, it has nothing to do with virt) at the new places and near future we have replace all KubeClient with controller-runtime client so we benefict from caching and from some test facilities it has (like deleting everything creating with it at `AfterSuite`)